pandas/arrays/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""
+All of pandas' ExtensionArrays and ExtensionDtypes.
+
+See :ref:`extending.extension-types` for more.
+"""
+from pandas.core.arrays import (
+    IntervalArray, PeriodArray, Categorical, SparseArray, IntegerArray,
+)
+
+
+__all__ = [
```

Comment:
we should consider renaming ``Categorical`` -> ``CategoricalArray`` (though deprecating maybe be very difficult)

Reply:
I think not worth it at this point.